akka-stream/src/main/scala/akka/stream/scaladsl/FlexiMerge.scala

```diff
@@ -233,4 +236,10 @@ abstract class FlexiMerge[Out, S <: Shape](val shape: S, attributes: OperationAt
     case Some(n) ⇒ n
     case None    ⇒ super.toString
   }
+
+  // FIXME what to do about this?
```

Comment:
Hmm, what a pickle. I’d say leave semantically as it is in this PR but change the UOE message to indicate that clients should override this method to make it work; then also add an example of how that looks like to the docs.